server/routerlicious/packages/services/src/redis.ts

```diff
@@ -3,20 +3,18 @@
  * Licensed under the MIT License.
  */
 
-import * as util from "util";
 import { ICache } from "@fluidframework/server-services-core";
-import { RedisClient } from "redis";
-
+import { Redis } from "ioredis";
 /**
  * Redis based cache client
  */
 export class RedisCache implements ICache {
     private readonly getAsync: any;
     private readonly setAsync: any;
 
-    constructor(client: RedisClient, private readonly prefix = "page") {
-        this.getAsync = util.promisify(client.get.bind(client));
-        this.setAsync = util.promisify(client.set.bind(client));
+    constructor(client: Redis, private readonly prefix = "page") {
+        this.getAsync = client.get.bind(client);
+        this.setAsync = client.set.bind(client);
```

Comment:
Do we want to add an error listener for the Redis client such as in server/routerlicious/packages/services/src/redisClientManager.ts?
```ts
client.on("error", (error) => {
            winston.error("Client Manager Redis Error:", error);
        });
```